apiserver/cache/redis.go

```diff
@@ -89,6 +90,19 @@ func (r *Cache) RemoveUserToken(ctx goContext.Context, token string) error {
 	return nil
 }
 
+func jtiKey(userId int, jti string) string {
+	return fmt.Sprintf("jti-%v-%v", strconv.Itoa(userId), jti)
+}
+
+func (r *Cache) AddJti(ctx goContext.Context, userId int, jti string, expiration time.Duration) (bool, error) {
```

Comment:
`SaveJti` maybe better name
and I would return only `error`, not `(bool, error)`